go.mod

```diff
@@ -1,156 +1,166 @@
 module github.com/scribd/go-sdk
 
-go 1.24
+go 1.24.0
```

Comment:
I believe this is incorrect. Shouldn't it still be `go 1.24` as we had before?

Reply:
@khaled-scribd No, this is correct. It is updated by the `go mod tidy`. One of the dependencies requires the `go v1.24.0` thus the update here.

Reply:
Got it, makes sense.